### DISCUSSION TITLE
Нужна ли инфа по брокерам сообщений?

### DISCUSSION THREAD

**Discussion by bzick** [bzick/oh-my-backend#8] [Ideas]
Большие кластера или сложные микросервисные проекты, используют брокеры сообщений для коммуникации или уведомлений. Но не ясно, стоит ли про них добавлять тут. Если стоит то в каком пункте? Или новый выделить на это?

Брокеры сообщений: Apache Kafka, RabbitMQ, Redis Streams

**Comment by Boring-Mind** [comment]
Ну тогда уже стоит и очереди добавить. Я бы добавил их рядом с микросервисной архитектурой, так как они, в основном, применяются там. Хотя можно и отдельную категорию сделать про распределенные системы, так как это отдельная область знаний, достойная упоминания. Тут решать вам.

**Reply by bzick** [reply]
Согласен что это пересекается с очередями. Пока не ясно в какой момент это лучше изучить что бы добавить в нужный/новый раздел
